### PR TITLE
Basic e2e tests framework

### PR DESCRIPTION
**Pull request by jakub-dzon**
framework package has been based on https://github.com/kubevirt/containerized-data-importer/blob/master/tests/framework/framework.go

```release-note
NONE
```

Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: tests/basic_vm_import_test.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
+	"github.com/kubevirt/vm-import-operator/tests/framework"
+	. "github.com/kubevirt/vm-import-operator/tests/matchers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TODO: for now running on machine with access to oVirt hosting cirrosvm VM is required.
```
Comment (pkliczewski) [inline tests/basic_vm_import_test.go]: Why not to use locally ovirt [mocks](https://github.com/kubevirt/vm-import-operator/pull/27/files#diff-a0068931f00eb90657e4a876bc012afbR1) that Ondra created some time ago.

File: tests/basic_vm_import_test.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			Fail("Cannot create secret: " + err.Error())
+		}
+		secret = s
+	})
+
+	Context(" without resource mapping", func() {
+		It("should create stopped VM", func() {
+			ovirtSecretName := secret.Name
+			// ID of a VM existing in oVirt described by the secret.
+			vmID := "c1a784d6-4747-4c26-8b47-d3d6766436e1"
```
Comment (pkliczewski) [inline tests/basic_vm_import_test.go]: If we decide to use Ondra's mocks this `ID` is OK but with real ovirt we would need to fetch this value.

File: tests/basic_vm_import_test.go (hunk 1 above)
Comment (jakub-dzon) [reply tests/basic_vm_import_test.go]: The automation env scripts are not merged yet. When I'm sure I can use them and mocks, I will use them. 
It's WIP and is only supposed to show general direction of the work.

File: tests/basic_vm_import_test.go (hunk 2 above)
Comment (jakub-dzon) [reply tests/basic_vm_import_test.go]: The desired state is having a mock that we can instruct to return specific objects.

**Review by pkliczewski — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Pull request merged**